src/Keywords/KeywordsInterface.php

```diff
@@ -14,6 +14,14 @@
  * Keywords holder interface.
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * @phpstan-type TLanguage string
```

Comment:
do we really need to create such alias instead of using `string` directly ?

Reply:
By itself it's pointless, but when used in other structures, it should be helpful.

For instance, `TMultiLanguageKeywords` could be defined as `array<string, ...>`, in which case I find it harder to convey that the key is the language.

Besides, it would make it easier to transition to finer-grained types in the future, if such a thing exists (e.g. if we're able to define the format of the string).